recipes/cpl/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# CPL's custom M4 macro that detects FFTW needs to find its shared library.
+# Putting the lib path directly in LD_LIBRARY_PATH does the trick,
+# and is actually recommended by the CPL documentation,
+# https://ftp.eso.org/pub/dfs/pipelines/libraries/cpl/docs/cpl-user-manual_6.0.pdf
+env LD_LIBRARY_PATH="${PREFIX}/lib:${LD_LIBRARY_PATH}" \
+  ./configure --prefix=${PREFIX}
+make
```

Comment:
Is there a `make check` we can run here?

Reply:
Yes, there is. I omitted it because it takes quite long to complete on my machine (approx. 3 minutes 45 seconds, vs 1m10s for `make`). If you say that's not too long @ocefpaf I'll add it.

Reply:
That is fine for our builds and the more tests the better :-)